### PR TITLE
Update qqmacmgr to 2.4.7

### PR DESCRIPTION
**Pull request by jinntrance**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by jinntrance** [comment]
Error occurred when installing qqmacmgr.
```
Error: Checksum for Cask 'qqmacmgr' does not match.
Expected: 50517f1fa76441886c1874c6dbfb2e910e3e2a20ad71316b7714485bb641c6cb
Actual:   142249e7567d41f84de53b30075e0f454377b3cf9635463dcd12e0381b83b199
File:     /Users/joseph/Library/Caches/Homebrew/Cask/qqmacmgr--2.4.1.dmg
```

File: Casks/qqmacmgr.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'qqmacmgr' do
   version '2.4.1'
```
Comment (gtmgianni) [inline Casks/qqmacmgr.rb]: The latest version of this application is `2.4.7`, not `2.4.1`.

Comment (gtmgianni) [inline Casks/qqmacmgr.rb]: The latest version of this application is `2.4.7`.

File: Casks/qqmacmgr.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-  sha256 '50517f1fa76441886c1874c6dbfb2e910e3e2a20ad71316b7714485bb641c6cb'
+  sha256 '142249e7567d41f84de53b30075e0f454377b3cf9635463dcd12e0381b83b199'
```
Comment (gtmgianni) [inline Casks/qqmacmgr.rb]: I've calculated the `sha256` for version `2.4.7` (not `2.4.1`) to be `ac3664a56667bd08d3538440507487ab0e0da70babdebe0eeb2c766c99c500bb`.

**Review by gtmgianni — CHANGES_REQUESTED** [review]

File: Casks/qqmacmgr.rb (hunk 2 above)
Comment (jinntrance) [reply Casks/qqmacmgr.rb]: Thank you. I've updated it to the latest version 2.4.7.

**Comment by jinntrance** [comment]
Seems Travis CI failed because of `Error: Cask 'abgx360' is unavailable: No Cask with this name exists.`.
After I try to install `abgx360` with the latest homebrew-cask on branch `master`, I got the error message: `Error: Cask 'abgx360' is unavailable: No Cask with this name exists. Did you mean “vin047-abgx360”?`.

**Comment by gtmgianni** [comment]
@jinntrance I believe you can ignore the result of Travis CI – it sometimes produces this error, although I'm not sure why (you'd need to ask one of the maintainers if this is a bug).

**Review by gtmgianni — APPROVED** [review]
```
1 file inspected, no offenses detected
==> Downloading http://dlied6.qq.com/invc/xfspeed/mac/verupdate/QQMacMgr_2.4.7.dmg
######################################################################## 100.0%
==> Verifying checksum for Cask qqmacmgr
audit for qqmacmgr: passed
```

**Comment by commitay** [comment]
The error is because your fork is outdated.

PRs should be submitted from a new branch of an up to date fork.

> https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#getting-set-up-to-contribute

I've cherrypicked the changes to https://github.com/caskroom/homebrew-cask/pull/41598

Thanks for the PR!

**Pull request closed without merging**